benchs/bench.inc

```diff
@@ -14,6 +14,12 @@ BUILD_IMAGES_DIR=$3
 BUILD_RESULTS_DIR=$4
 DATE=$5
 
+# Optional parameter. Not all the benchmarks use this.
+VM_PAR_EDEN_SIZE=
```

Comment:
Why VM_PAR_EDEN_SIZE and not just VM_PARAMETERS?
Because what you do sends just a string as VM parameter, right?

Reply:
You can use the syntax ${ITERATIONS:=1} to handle default values

Reply:
Done :)